lib/Clownface.js

```diff
@@ -252,12 +252,14 @@ class Clownface {
   /**
    * Creates a graph pointer to nodes which link the current pointer by `predicates`
    * @param {Term|Term[]|Clownface|Clownface[]} predicates one or more RDF/JS term identifying a property
+   * @param {object} [options]
+   * @param {string | string[] | undefined} [options.language]
    * @returns {Clownface}
    */
-  out (predicates) {
+  out (predicates, options) {
```

Comment:
A destructuring argument could make the interface more readable.

Reply:
Actually in this location I thought a standard variable parameter more clean when it is just passed as a whole to the `current.out()` call below rather then repeating all properties.